### PR TITLE
Traceback création tâche Dispatch

### PR DESCRIPTION
**Pull request by ventilooo**
* Add `False` as default value for user_id
* Set html field content to `utf-8`

### CODE REVIEW

File: project_task_autocreate/models/project_task_template.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def get_task_vals(self, record=None):
             'priority': self.priority,
             'tag_ids': [(6, 0, self.tag_ids.ids)],
             'date_deadline': self._get_deadline(record),
-            'user_id': self._get_user_id(record).id,
+            'user_id': self._get_user_id(record),
```
Comment (joaoalf) [inline project_task_autocreate/models/project_task_template.py]: @ventilooo If you look carefully at the `get_user_id`, `get_partner_id` and `get_project_id` you will see that they all return a record. Keep that consistency.

**Comment by ghost** [comment]
@ventilooo this can be simplified. Also, you should also fix _get_partner_id and _get_project_id.

**Review by rimbendhaou — COMMENTED** [review]
@ventilooo If description contains empty fields values it displays False. Is it accepted? It is better to not display False in the discription (just put empty values)

**Comment by rimbendhaou** [comment]
@ventilooo Is it possible to have validation of the description to check its value before saving the task template?

**Comment by ventilooo** [comment]
@joaoalf I'd like your opinion on @rimbendhaou feedbacks.
* Since `description` is an HTML field, how hard would it be to add validation ? and what should be considered valid ?
* For the `False` , I suppose that once the field will be filled with a correct value the false displayed in the task name/description will be overwrite?

**Review by ghost — APPROVED** [review]

**Pull request merged**